x/lightclient/ante/ibc_msgs_test.go

```diff
@@ -17,6 +17,11 @@ type MockRollappKeeper struct {
 	stateInfos map[string]map[uint64]rollapptypes.StateInfo
 }
 
+func (m *MockRollappKeeper) GetLatestHeight(ctx sdk.Context, rollappId string) (uint64, bool) {
+	// TODO implement me
```

Comment:
I guess no need this line

Reply:
it's autogenerated, gonna dismiss as nit, but good point 👍